econet_electric_tank_water_heater.yaml

```diff
@@ -57,16 +61,19 @@ sensor:
     sensor_datapoint: POWRWATT
     unit_of_measurement: "W"
     accuracy_decimals: 1
-    device_class: "power"
```

Comment:
Keep this here and add the filters at the end. It seems most if not all of existing filters are at the end. For consistency add them at the end and to minimize diff like here.